frontend/src/components/key-value-table/key-value-table.tsx

```diff
@@ -0,0 +1,32 @@
+import type { HTMLAttributes, ReactNode } from 'react'
+
+import { Colors } from '../../config/colors'
+
+interface KeyValueTableProps extends HTMLAttributes<HTMLTableElement> {
+  rows: Array<{ key: ReactNode; value: ReactNode; dataTestId?: string }>
+}
+
+export const KeyValueTable = ({ rows, style }: KeyValueTableProps) => {
```

Comment:
comment (non-blocking): the more we go into this the more I think we should get UI toolkit publishing as soon as we can

Reply:
Or alternatively get the desktop wallet UI into the monorepo, up for discussion